invoices_test.go

```diff
@@ -21,6 +21,8 @@ const oneInvoiceExample = `{
 			"external_id": null,
 			"type": "subscription",
 			"subscription_uuid": "sub_e6bc5407-e258-4de0-bb43-61faaf062035",
+			"subscription_external_id": "sub_external_id_123",
```

Comment:
What are the empty values? Can you add a test case for that?
Hint: it can be either missing field/nil/empty string, depending on how Import API behaves.